Update information about MSI bug (#24595)
docker/docker.github.io: content/manuals/desktop/release-notes.md

#### For Windows

- Fixed high CPU usage in the Windows API proxy caused by unnecessary process enumeration.

> - Fixed the Windows MSI installer failing to update Docker Desktop. Versions between 4.56 and 4.65 need to uninstall before reinstalling a version 4.66 or later. Note that uninstalling removes all associated data.

Reviewer: ```suggestion
- Fixed the Windows MSI installer failing to update Docker Desktop. Versions between 4.56 and 4.65 need to uninstall before reinstalling version 4.66 or later. Note that uninstalling removes all associated data.
```

### Known issues

> - The Windows MSI installer cannot update an existing Docker Desktop installation when the current version is between 4.56 and 4.65. As a workaround, uninstall the existing version before reinstalling. Note that uninstalling removes all associated data. A fix will be provided in a future MSI release.

Reviewer: ```suggestion
- The Windows MSI installer cannot update an existing Docker Desktop installation when the current version is between 4.56 and 4.65. As a workaround, uninstall the existing version before reinstalling the latest version. Note that uninstalling removes all associated data. 
```
